dm/syncer/data_validator.go

```diff
@@ -777,3 +783,35 @@ func genColData(v interface{}) string {
 	s := fmt.Sprintf("%v", v)
 	return s
 }
+
+func (v *DataValidator) GetValidationStatus() []*pb.ValidationStatus {
+	failpoint.Inject("MockValidationQuery", func() {
+		failpoint.Return([]*pb.ValidationStatus{
+			{
+				SrcTable:         "`testdb1`.`testtable1`",
+				DstTable:         "`dstdb`.`dsttable`",
+				ValidationStatus: pb.Stage_Running.String(),
+			},
+			{
+				SrcTable:         "`testdb2`.`testtable2`",
+				DstTable:         "`dstdb`.`dsttable`",
+				ValidationStatus: pb.Stage_Stopped.String(),
+				Message:          tableWithoutPrimaryKeyMsg,
+			},
+		})
+	})
+	ret, err := v.persistHelper.loadTableStatus(v.tctx)
+	if err != nil {
+		return []*pb.ValidationStatus{}
+	}
+	result := make([]*pb.ValidationStatus, 0)
+	for _, tblStat := range ret {
+		result = append(result, &pb.ValidationStatus{
+			SrcTable:         dbutil.TableName(tblStat.source.Schema, tblStat.source.Name),
+			DstTable:         dbutil.TableName(tblStat.target.Schema, tblStat.target.Name),
```

Comment:
```suggestion
			SrcTable:         tblStat.source.String(),
			DstTable:         tblStat.target.String(),
```


It is better to unify `DstTable` and `target`.

And a suggestion is that `tblStat.source` and `tblStat.target` also should add a suffix such as `Table` to show their identity in naming.

Reply:
Will resolve it.

Naming issue: /cc @D3Hunter

Reply:
since the struct is named `tableValidateStatus`, it's ok to omit `table` suffix i think